com.unity.render-pipelines.high-definition/Runtime/Sky/SkyUtils.hlsl

```diff
@@ -17,4 +17,14 @@ float3 GetSkyViewDirWS(float2 positionCS)
     return normalize(viewDirWS.xyz);
 }
 
+// Returns latlong coords from view direction
+float2 GetLatLongCoords(float3 dir, float upperHemisphereOnly)
+{
+    const float2 invAtan = float2(0.1591, 0.3183);
+    float2 uv = float2(atan2(dir.x, dir.z), asin(dir.y));
```

Comment:
Please use fastAtan / fastAsin (see common.Hlsl), also see if they have the required precision (try with a 4K sky to check that we don't get artifact)

Reply:
I am not too sure about my use of fastAtan for atan2 but it works with minor precision difference